docs/getting-started/configuring-your-repository.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Configure the Codacy analysis tools to match the coding conventions and best practices that your team may already be following or that you want to promote.
```

Comment:
Are we configuring the "repository" or the "analysis tool"? I'm seeing both expressions used across this content. Do we need to clarify this? 🤔

Reply:
Good point. I tweaked the wording